bot/sx-bot.el

```diff
@@ -0,0 +1,82 @@
+;;; sx-bot.el --- Functions for viewing different tabs.       -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2014  Artur Malabarba
+
+;; Author: Artur Malabarba <bruce.connor.am@gmail.com>
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;
+
+
+;;; Code:
+
+(require 'sx-site)
+(require 'sx-tag)
+
+(setq sx-request-remaining-api-requests-message-threshold 50000)
+
+(require 'package)
+(package-initialize)
+
+(defcustom sx-bot-out-dir "./data/tags/"
+  "Directory where output tag files are saved."
+  :type 'directory
+  :group 'sx)
+
+
+;;; Printing
+(defun sx-bot-write-to-file (data)
+  "Write (cdr DATA) to file named (car DATA).
+File is savedd in `sx-bot-out-dir'."
+  (let ((file-name (expand-file-name (car data) sx-bot-out-dir)))
+    (with-temp-file file-name
+      (let* (print-length
+             (repr (prin1-to-string
```

Comment:
Another option: we could set up some sort of cron job on either of our servers and download the archive from there. It's not as 'clean' as coming straight from GitHub, though.

I'm looking into any possibility of keeping the archive the 'front matter' file of the git repository, but keeping the plain text in the 'back matter'. This would obviously be the best of both worlds: small diffs, official distribution.